README.md

```diff
@@ -10,6 +10,8 @@ A project scaffold for building [Ghost](http://github.com/tryghost/ghost/) theme
 
 - Blazing fast Lib-Sass
 
+- Bourbon and Neat
+
```

Comment:
Thoughts on cutting the empty lines between each list item? It outputs unnecessary `p` tags inside each item.